### PR TITLE
news: Accept first customers

### PR DESCRIPTION
**Pull request by ZJvandeWeg**

### CODE REVIEW

File: src/blog/flowforge-accepting-customers.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+when that was the case, we [opened up our waitlist](...). The waitlist is growing
+daily, a trend we're aiming to break today. The first lucky few users are added
+today, and each working day for the foreseeable future new ones are added. New
+users will receive an email from our team with their login details and can start
+creating new workflows with Node-RED minutes after.
+
+While the source code of FlowForge is [available](https://github.com/flowforge/flowforge),
+there's a chance you're unfamiliar with what has been built around Node-RED.
+With FlowForge, our intent is to build a platform to aid with colaboration of
+flows for any Node-RED. The first steps to our vision include a managed
```
Comment (sammachin) [inline src/blog/flowforge-accepting-customers.md:26]:
```suggestion
flows in Node-RED. The first steps to our vision include a managed
```

File: src/blog/flowforge-accepting-customers.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+Node-RED instance to connect virtually any online service with another with the
+interface already knows. Multiple users will have access to the same flows and
+can collaborate. Further, once Node-RED 3.0 will be release, upgrading will take
+only a single click. There's many more exciting features right around the corner
+on our [roadmap](https://github.com/orgs/flowforge/projects/5).
```
Comment (sammachin) [inline src/blog/flowforge-accepting-customers.md:31]: Roadmap isn't currently public, but I'm happy to open it up

Comment (ZJvandeWeg) [reply src/blog/flowforge-accepting-customers.md:31]: Didn't consider that, but would be nice to open up.

File: src/blog/flowforge-accepting-customers.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+today, and each working day for the foreseeable future new ones are added. New
+users will receive an email from our team with their login details and can start
+creating new workflows with Node-RED minutes after.
+
+While the source code of FlowForge is [available](https://github.com/flowforge/flowforge),
+there's a chance you're unfamiliar with what has been built around Node-RED.
+With FlowForge, our intent is to build a platform to aid with colaboration of
+flows for any Node-RED. The first steps to our vision include a managed
+Node-RED instance to connect virtually any online service with another with the
+interface already knows. Multiple users will have access to the same flows and
```
Comment (sammachin) [inline src/blog/flowforge-accepting-customers.md:28]: Delete `with the
interface already knows.`
It doesn't make sense

File: src/blog/flowforge-accepting-customers.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+can collaborate. Further, once Node-RED 3.0 will be release, upgrading will take
```
Comment (sammachin) [inline src/blog/flowforge-accepting-customers.md:29]:
```suggestion
can collaborate. Further, once Node-RED 3.0 has been released, upgrading will take
```

**Comment by knolleary** [comment]
With the recent reorganisation of the blog posts, can you move this file to: `src/blog/2022/04/flowforge-accepting-customers.md` ... although you may want to do that *after* resolving any items on the content as I don't know how GH copes with review commends on moved files...

File: src/blog/flowforge-accepting-customers.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
+---
+title: FlowForge is accepting customers now
+subtitle: From today forward, we start accepting customers of our waitlist
+description: From today forward, we start accepting customers of our waitlist
+date: 2022-04-04
+authors: ["zeger-jan-van-de-weg"]
+---
+
+A year ago our CTO, Nick O'Leary, [introduced FLowForge](https://flowforge.com/blog/2021/04/first-deploy/).
```
Comment (knolleary) [inline src/blog/flowforge-accepting-customers.md:9]:
```suggestion
A year ago our CTO, Nick O'Leary, [introduced FlowForge](https://flowforge.com/blog/2021/04/first-deploy/).
```

File: src/blog/flowforge-accepting-customers.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+Since then major milestones have been achieved. As we grow as a company, important
+steps are taken. Today we make another very important step; we're accepting our
+first customers.
+
+<!--more-->
+
+A few weeks ago our product was nearly in a state where it could support customers,
+when that was the case, we [opened up our waitlist](...). The waitlist is growing
+daily, a trend we're aiming to break today. The first lucky few users are added
+today, and each working day for the foreseeable future new ones are added. New
```
Comment (knolleary) [inline src/blog/flowforge-accepting-customers.md:19]:
```suggestion
daily, a trend we're aiming to break today. The first lucky few users are being added
today, and each working day for the foreseeable future we'll continue onboarding users. New
```

File: src/blog/2022/04/flowforge-accepting-customers.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+A year ago our CTO, Nick O'Leary, [introduced FlowForge](https://flowforge.com/blog/2021/04/first-deploy/).
+Since then major milestones have been achieved. As we grow as a company, important
+steps are taken. Today we make another very important step; we're accepting our
+first customers.
+
+<!--more-->
+
+A few weeks ago our product was nearly in a state where it could support customers,
+when that was the case, we [opened up our waitlist](...). The waitlist is growing
```
Comment (knolleary) [inline src/blog/2022/04/flowforge-accepting-customers.md:17]:
```suggestion
when that was the case, we [opened up our waitlist](https://flowforge.com/blog/2022/02/announcing-flowforge-cloud/). The waitlist is growing
```

This link is to the original announce post - probably best place to link to the waitlist.

**Pull request merged**